loomi/loomi-ui-flutter
# feature/example-app-and-storybook

_Changes:_

_1. What i did:_

- Fix CustomButton Component
- Fix TextFormField Component
- Create DefaultLoader Component
- Crete Example App
  - Create Example Screen with components
  - Create StoryBook section to check Components Options

_2. What to test:_

- [ ] Check the components usage
- [ ] Check Example Screen
- [ ] CodeReview

---

_How to get there_

_1. Flavor and login informations_

Not needed

_2. Steps_

- Run the app
- Choose Example or StoryBook Button
---

_Observations:_

_1. Known issues_

- Not a issue, but StoryBook only works when running on Web.

_2. Code review pratices_

- Unecessary prints/log (ctrl+f -> [ print( ] )
- Unecessary comments (ctrl+f -> [ // ] )
- Code organization
- Naming of functions and variables
- Repeated code
